src/jit/simd.cpp

```diff
@@ -351,124 +351,136 @@ var_types Compiler::getBaseTypeAndSizeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd, u
 #if FEATURE_HW_INTRINSICS
     else if (isIntrinsicType(typeHnd))
     {
+        enum
+        {
+            Vector256Size = 256 / 8,
+            Vector128Size = 128 / 8,
+        };
```

Comment:
We can define `Vector128/256Size` as `const` or `constexpr` (for `static_assert`) rather than `enum`.

Reply:
I preferred an enum because they are logically grouped that way.  Either way is fine by me.  I'll let the code owners decide.

Reply:
Typically smaller values precede larger values, so:

```
           Vector128Size = 128 / 8,
           Vector256Size = 256 / 8,
```

Reply:
Personally I would prefer to see these as `const` rather than `enum`, because I don't think of these as being enumerated values. Also, their name should indicate the units, e.g. `Vector128Bytes`.